### ISSUE TITLE
Instructions for replicating font settings

### ISSUE THREAD

**Issue by MatthewScholefield** [kcamcam/vscode_dark_modern.zed#4]
In case anyone else wants to get the font rendering as close to VSCode as possible, these are the settings I arrived upon that I found matched closest:

```json
"ui_font_size": 16,
"buffer_font_size": 13.7037024497986,
"buffer_font_family": "Droid Sans Mono",
"buffer_font_weight": 100,
"buffer_line_height": {
    "custom": 1.35
},
```

And I made sure to install the [Droid Sans Mono](https://www.fontsquirrel.com/fonts/droid-sans-mono) font on my system and restart Zed to get it to pick it up. For Linux:

```bash
mv DroidSansMono.ttf ~/.local/share/fonts/
rm -r ~/.cache/fontconfig/
fc-cache -r
```

**Comment by MatthewScholefield** [comment]
Yeah. @2zqa Curious, do you think no line highlight is better than having a very subtle line highlight color like that used in the new proposed colors?

Personally I think having a really subtle background color change for the line highlight feels similar to VSCode's top and bottom thin line indicators.

**Comment by 2zqa** [comment]
That's a good one, I didn't even consider that option. Good idea.

**Comment by 2zqa** [comment]
Perhaps we can add this to the README.md. @MatthewScholefield, do you want to create a PR for this?
